ci-operator/jobs/infra-periodics.yaml

```diff
@@ -308,6 +308,63 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-bot
+- agent: kubernetes
+  labels:
+    ci.openshift.io/role: infra
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  cron: "30 10 * * 2"  # Run at 10:30 UTC each Tuesday
+  decorate: true
+  name: periodic-prow-auto-config-brancher
+  spec:
+    containers:
+    - command: ["/bin/bash"]
+      args:
+      - |
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+        echo "machine github.com" > ~/.netrc
+        echo "login ${GIT_COMMITTER_NAME}" >> ~/.netrc
+        echo "password token" >> ~/.netrc
+        ### Be CAREFUL ... potential leaking?
+        sed -e "s/token/$(cat /etc/github/token)/" ~/.netrc
+        /user/bin/determinize-ci-operator --config-dir ./ci-operator/config --current-release ${CURRENT_RELEASE} --future-release ${FUTURE_RELEASE} --confirm
+        /user/bin/config-brancher --config-dir ./ci-operator/config --current-release ${CURRENT_RELEASE} --future-release ${FUTURE_RELEASE} --confirm
+        /user/bin/ci-operator-prowgen --from-dir ./ci-operator/config --to-dir ./ci-operator/jobs
+        git push "https://github.com/${GIT_COMMITTER_NAME}/release.git" HEAD:auto-config-brancher -f
+        # TODO: ENSURE_PR
```

Comment:
If this means you need Go code, then we can run the above script inside of it (or before it) :(